### PR TITLE
Add an algorithm to find length of an arc and area of the sector formed by an arc of a circle

### PR DESCRIPTION
**Pull request by prashalruchiranga**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by appgurueu — APPROVED** [review]
Simple enough that it doesn't need tests.

File: Maths/CircularArc.js [hunk 1]
```diff
@@ -6,36 +6,36 @@
...
+  /**
+   * @function circularArcLength
+   * @description calculate the length of a circular arc
+   * @param {Integer} radius
+   * @param {Integer} degrees
+   * @returns {Integer} radius * angle_in _adians
+   * @see https://en.wikipedia.org/wiki/Circular_arc
+   * @example circularArcLength(3, 45) = 2.356194490192345
+   */
+  function circularArcLength (radius, degrees) {
```
Comment (appgurueu) [inline Maths/CircularArc.js]: why are you indenting the functions now?

Comment (prashalruchiranga) [reply Maths/CircularArc.js]: I thought it would be better if the code followed the standard style

Comment (appgurueu) [reply Maths/CircularArc.js]: The standard style mandates indenting top level functions? I doubt that.

File: Maths/CircularArc.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+function degreesToRadians (degrees) {
+  return degrees * Math.PI / 180
+}
+
+/**
+   * @function circularArcLength
+   * @description calculate the length of a circular arc
+   * @param {Integer} radius
+   * @param {Integer} degrees
+   * @returns {Integer} radius * angle_in _adians
```
Comment (appgurueu) [inline Maths/CircularArc.js:18]: angle_in_radians*

**Review by appgurueu — DISMISSED** [review]

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
Degrees to Radians is also an useful algorithm. Put it in a separate file and export it. Also add tests for all the algorithms.

**Comment by prashalruchiranga** [comment]
@raklaptudirm  I have exported the Degrees to Radians function from an already available file and added tests as you requested.

File: Maths/test/CircularArc.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { circularArcLength } from "../CircularArc";
```
Comment (appgurueu) [inline Maths/test/CircularArc.test.js]:
```suggestion
import { circularArcLength, circularArcArea } from "../CircularArc";
```

**Review by appgurueu — APPROVED** [review]

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**